src/main/java/com/faforever/moderatorclient/mapstruct/MapMapper.java

```diff
@@ -6,7 +6,7 @@
 
 import java.util.List;
 
-@Mapper(uses = {JavaFXMapper.class, MapVersionMapper.class, PlayerMapper.class, CycleAvoidingMappingContext.class})
+@Mapper(componentModel = "spring", uses = {JavaFXMapper.class, MapVersionMapper.class, PlayerMapper.class, CycleAvoidingMappingContext.class})
```

Comment:
componentModel = "spring" ... what is that for

Reply:
telling spring that this is a component